src/drawings/circle/Circle.jsx

```diff
@@ -3,26 +3,26 @@
 import { CanvasBuilder } from '@/components/Canvas'
 import { Color } from '@/constants'
 
+const HEIGHT = 100
+const WIDTH = 100
 const initialState = {
   i: 0,
   delta: 0.05
 }
 
 /** @type {import('@/components/Canvas/CanvasBuilder').DrawFactory} */
-function drawAnimatedCircle (canvas, { drawState, setDrawState }) {
+function drawAnimatedCircle (ctx, { drawState, setDrawState }) {
   let requestId
   let { i, delta } = drawState
 
   /** @type {import('@/components/Canvas/CanvasBuilder').DrawFunction} */
   function draw ({ isPaused }) {
-    const ctx = canvas.getContext('2d')
-
-    ctx.clearRect(0, 0, canvas.width, canvas.height)
```

Comment:
We can use the constants `HEIGHT` and `WIDTH` here too. @belcurv make this change and test